processing-tests/common/test/kotlin/com/livefront/sealedenum/compilation/visibility/ProtectedInterfaceSealedClassWithDifferentPackageBaseClassTests.kt

```diff
@@ -47,7 +48,12 @@ class ProtectedInterfaceSealedClassWithDifferentPackageBaseClassTests {
         )
     }
 
+    /**
+     * TODO: kotlin-compile-testing seems to only allow KSP to view Java files with an Origin of CLASS, which causes
+     *       issues when trying to look at the PROTECTED modifier. Per above, in real use this usage still compiles.
+     */
```

Comment:
I should double check that disabling this is still required with the latest version of `kotlin-compile-testing`, some things might have changed slightly since.

It does work fine during the normal compilation process, so there might need to be an issue opened on `kotlin-compile-testing`.